tests/cue/shared_fs.py

```diff
@@ -0,0 +1,93 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), 'src'))
+from shared_fs_list import shared_fs, shared_fs_sites
+
+
+@rfm.simple_test
+class VSCSharedFSMountTest(rfm.RunOnlyRegressionTest):
+    descr = "test shared filesystem mount point "
+    fs = parameter(shared_fs)
+    site = parameter(shared_fs_sites)
+    valid_systems = ["*:local", "*:single-node"]
+    valid_prog_environs = ["builtin"]
+    maintainers = ['rverschoren']
+    time_limit = '10m'
+    num_tasks = 1
+    num_tasks_per_node = 1
+    num_cpus_per_task = 1
+    tags = {"antwerp"}
+
+    @run_after('init')
+    def set_param(self):
+        path = os.path.join(self.fs['mount'], self.site)
+        self.descr += path
+        exe = """python3 -c 'import os;print(os.path.isdir(os.path.realpath("{}")))'"""
+        self.executable = exe.format(path)
+
+    @sanity_function
+    def assert_env(self):
+        return sn.assert_found(r'^True$', self.stdout)
+
+
+@rfm.simple_test
+class VSCSharedFSMode(rfm.RunOnlyRegressionTest):
+    descr = "test shared filesystem mode "
+    fs = parameter(shared_fs)
+    site = parameter(shared_fs_sites)
+    valid_systems = ["*:local", "*:single-node"]
+    valid_prog_environs = ["builtin"]
+    maintainers = ['rverschoren']
+    time_limit = '10m'
+    num_tasks = 1
+    num_tasks_per_node = 1
+    num_cpus_per_task = 1
+    tags = {"antwerp"}
+
+    @run_after('init')
+    def set_param(self):
+        path = os.path.join(self.fs['mount'], self.site)
+        self.descr += path
+        mode = self.fs.get('mode')
+        if not mode:
+            # default: check if directory has rwxr-xr-x permissions 
+            mode = '755'
```

Comment:
should we also check the mode of $VSC_{HOME,DATA,SCRATCH}?
in Hydra, the mode is 700

Reply:
Checking $VSC_{HOME,DATA,SCRATCH} permissions is postponed until further discussion. 
This first iteration of the tests should check availability of the directories, which should work in this PR.